File: OJ/Luogu/P3150.js
function P3150(n) {
    if (n % 2 === 0) {
        console.log("pb wins");
    } else {
        console.log("zs wins");
    }
}
let a = [1, 3, 7, 20, 5]
for (let i = 0;i < 5; i ++) {
    P3150(a[i])
}

File: OJ/Luogu/P3150.cpp
#include<iostream>
#include<cstdio>
#include<cmath>

using namespace std;
int main() {
  int n;
  cin >> n;
  while(n--) {
    int temp;
    cin >> temp;
    if (temp % 2 == 0) {
      cout << "pb wins" << endl;
    } else {
      cout << "zs wins" << endl;
    }
  }
  return 0;
}